### PR TITLE
🪲 Fix make_response incorrectly sending `text` instead of JSON

### LINKED ISSUE #5550: 🪲 Console errors related to user activity
This is happening on `alpha` right now, we are getting console errors related to some JSON. I checked the response that is being turned to json and it seemed fine, does anyone else have any clue?

image

### PR DESCRIPTION
**Pull request by jpelay**
#5558 Introduced an error where some calls where expecting a JSON, and got text instead. I checked the places where this happened and fix the calls. 

If I'm missing some, please add them 😄 

**How to test**
Perform the following: 

* Update an adventure
* Update  a class customization
* Create students account
* Reset password
* Reset a student's password
* Update your public profile

And all of them should show the success modal! 

Also:
Fixes #5550

### CODE REVIEW

**Comment by Annelein** [comment]
We've found another error, I will solve this tomorrow!

**Comment by jpelay** [comment]
@Annelein I fixed the achievements issue! Can you check again please?

**Comment by Annelein** [comment]
@jpelay I've changed some 204's to 200's, because the `make_response` might not have content, but there might be a check for `response.achievements` on the frontend, which will result in an error if it's a 204.

Only comment left is about `response` instead of `response_body`, otherwise I will approve!

**Comment by jpelay** [comment]
> @jpelay I've changed some 204's to 200's, because the `make_response` might not have content, but there might be a check for `response.achievements` on the frontend, which will result in an error if it's a 204.

Perfect, thanks!

> Only comment left is about `response` instead of `response_body`, otherwise I will approve!

What's the comment?

File: website/classes.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def create_class(self, user):
 
         self.db.store_class(Class)
         achievement = self.achievements.add_single_achievement(user["username"], "ready_set_education")
-        if achievement:
-            utils.add_pending_achievement({"achievement": achievement})
-        return make_response({"id": Class["id"]}, 200)
+        response_body = {"id": Class["id"]}
```
[the remark this answers is not in the document]
Comment (jpelay) [reply website/classes.py:58]: I prefer response too! But in some functions there's a `response` variable already set, and I prefer to make a new one.

[the remark this answers is not in the document]
Comment (Annelein) [reply website/classes.py:58]: true! But not for this function right? And what about the others?

[the remark this answers is not in the document]
Comment (jpelay) [reply website/classes.py]: You're right, just renamed them :D

**Review by Annelein — APPROVED** [review]

**Pull request merged**